modules/widgets/mailchimp/css/admin.css

```diff
@@ -0,0 +1,170 @@
+.mailchimp-widget-jetpack-form-wrapper {
+    margin: 0 -15px 0 -15px;
```

Comment:
Could you switch from spaces to tabs here and elsewhere in this PR?

In general, I would recommend setting your IDE so spaces are converted to tabs on save if possible.

Reply:
@jeherve Thanks for noticing that, this is fixed in the last commit.

Reply:
Thank you! I'd recommend fixing the other CSS files as well.

Reply:
Thanks @jeherve I just switched to tabs for all css files in the widget, it seems I did misread your suggestion, I apologize :+1: